packages/ui/src/ui/utils/tablet_cell_bundles/bundles/bundle-editor-dialog/index.ts

```diff
@@ -269,6 +270,25 @@ export const simpleBundleValidate = (v?: number | string) => {
     return error;
 };
 
+export const dcBundleValidate = ({
+    length,
+}: BundleDataCenter[]): ((input?: number | string) => string | undefined) => {
+    return (input?: number | string): string | undefined => {
+        const number = Number(input);
+        let error: string | undefined;
+
+        if (!Number.isFinite(number)) {
+            error = 'Incorrect value';
+        } else if (number < 0) {
+            error = 'Must be greater than or equal to zero';
+        } else if (length > 0 && number % length !== 0) {
+            error = `Must divide the number of data centers (${length})`;
```

Comment:
suggestion: Looks like this is not worded correctly. I would recommend changing for:
`Must be a multiple of the number of data centers (${length})`